scripts/scil_compute_fodf_max_in_ventricles.py

```diff
@@ -2,7 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Script to compute the maximum fODF in the ventricles.
+Script to compute the maximum fODF in the ventricles. The ventricules are
+estimated from a MD and FA threshold.
+
+This allows to clip the noise of fODF using an absolute thresold.
```

Comment:
We should cite: 
Dell’acqua, F., Simmons, A., Williams, S. C. R., & Catani, M. (2012). Can spherical deconvolution provide more information than fiber orientations? Hindrance modulated orientational anisotropy, a true-tract specific index to characterize white matter diffusion. Human Brain Mapping.